aces/Loris
# [php/libraries/NDB_Menu_Filter.class.inc] filter undefined fix

## Brief summary of changes

Fixes this error I received today when visiting the user_accounts module.
`Notice: Undefined index: filter in /Users/alizee/Development/GitHub/McGill/Loris/php/libraries/NDB_Menu_Filter.class.inc on line 598`

#### Testing instructions (if applicable)

1. Visit the user_accounts module on fresh rasinbread installation with only admin account.
2. Check error_log
